### DISCUSSION TITLE
Ussing function buttons on stylus while drawing

### DISCUSSION THREAD

**Discussion by chepalitos** [xournalpp/xournalpp#2615] [Ideas] [answered]
Hi. Not sure if here is the right place to comment.
I am ussing stylus from a wacom tablet. When using the stylus buttons, they start working fine with the pencil lifted from tablet. While this buttons are been press, if stylus is touching the drawing surface it start drawing. Any chance to use the functions on the pencil buttons while it is touching the surface?

**Comment by chepalitos** [comment, marked as answer]
Hm... I found I have this issu in general. I thought it was jus with this app, wich is the one I use the most =)

**Reply by LittleHuba** [reply]
You can try to enable (or disable) merging of tip events with button events in the preferences of Xournal++. I'm not quite sure it will help in this instance but it is worth a try!
